### PR TITLE
Added capability to restart browser if it wasn't open, version bumped

### PR DESCRIPTION
**Pull request by georgiy-abbasov**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -74,7 +83,9 @@ export default class BrowserStackConnector {
         return worker && worker.browser_url;
     }
 
-    async startBrowser (browserSettings, url, { jobName, build } = {}, timeout = null) {
+    async _startBrowser (browserSettings, url, { jobName, build }, timeout) {
+        const urlIdentifier = uid(10);
```
Comment (AlexanderMoskovkin) [inline src/index.js]: `browserId`

File: src/index.js [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
 import { createClient } from 'browserstack';
 import { Local as BrowserStackLocal } from 'browserstack-local';
+import uid from 'uid';
 import wait from './utils/wait';
+import Hub from './hub.js';
+
+const START_BROWSER_MAX_ATTEMPT = 3;
+const START_BROWSER_TIMEOUT     = 60 * 1000;
+
+const HUB_PORT = 1000;
```
Comment (AlexanderMoskovkin) [inline src/index.js]: Move this settings to options:

`HUB_PORT` -> to BrowserStackConnector constructor options as `servicePort`

Others to:
`startBrowser (browserSettings, url, { jobName, build } = {}, { maxAttepts, openingTimeout, workingTimeout } = {})`

File: src/index.js [hunk 3]
```diff
@@ -105,11 +116,60 @@ export default class BrowserStackConnector {
...
+        let error   = null;
+
+        while (attempt < START_BROWSER_MAX_ATTEMPT && !worker) {
+            try {
+                error  = null;
+                worker = await this._startBrowser(browserSettings, url, { jobName, build }, timeout);
+            }
+            catch (err) {
+                error  = err;
+                worker = null;
```
Comment (AlexanderMoskovkin) [inline src/index.js]: it's not necessary to set `worker = null` here because it's `null` if `this._startBrowser()` failed

**Review by AlexanderMoskovkin — CHANGES_REQUESTED** [review]

**Comment by georgiy-abbasov** [comment]
FPR

**Review by AlexanderMoskovkin — APPROVED** [review]

**Pull request merged**